### PR TITLE
corrected usage of articles in readme

### PR DESCRIPTION
**Pull request by KrishnaMoorthy12**
I just found a few English grammar errors in the README. I noticed that they're fine in the docs. I didn't open any issue regarding this because I thought it didn't need that much attention. 

P.S: It's okay if you think this PR is worthless.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Typegoose
 
-<sub>(These badges are from typegoose:master)</sub>  
+<sub>(These badges are from typegoose:master)</sub>
```
Comment (hasezoey) [inline README.md:3]: when possible, please revert this change

**Review by hasezoey — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KrishnaMoorthy12) [reply README.md:3]: Added a line break there to get the badges down to the next line. It seems prettier removed the trailing white space.
I'm not sure how the trailing white space got the badges to the next line earlier.

Comment (hasezoey) [reply README.md:3]: > I'm not sure how the trailing white space got the badges to the next line earlier.

in markdown you can do an line-break in 3 way:
- add another empty line (not preferred)
- add raw html `<br/>`
- add 2 spaces at the end of the line (which was here previously)

Comment (KrishnaMoorthy12) [reply README.md:3]: Oh, I didn't know the third way.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
```
Comment (hasezoey) [inline README.md:3]: please convert this to the double space as it was before

File: README.md [hunk 3]
```diff
@@ -35,16 +36,20 @@ const UserModel = getModelForClass(User); // UserModel is a regular Mongoose Mod
...
+model changes, you also have to keep the TypeScript interface file in sync or the TypeScript interface would not represent the real data
+structure of the model.
 
-Typegoose aims to solve this problem by defining only a TypeScript interface (class), which needs to be enhanced with special Typegoose decorators (like `@prop`).
+Typegoose aims to solve this problem by defining only a TypeScript interface (class), which needs to be enhanced with special Typegoose
+decorators (like `@prop`).
 
-Under the hood it uses the Reflect & [reflect-metadata](https://github.com/rbuckton/reflect-metadata) API to retrieve the types of the properties, so redundancy can be significantly reduced.
+Under the hood it uses the Reflect & [reflect-metadata](https://github.com/rbuckton/reflect-metadata) API to retrieve the types of the
+properties, so redundancy can be significantly reduced.
```
Comment (hasezoey) [inline README.md:47]: please get them on the same lines as it was before

File: README.md [hunk 4]
```diff
@@ -149,8 +154,8 @@ To ask questions or just talk with us, [join our Discord Server](https://discord
 
 ## Documentation
 
-* [Typegoose Documentation](https://typegoose.github.io/typegoose/docs/api/index-api)
-* [Quick start guide](https://typegoose.github.io/typegoose/docs/guides/quick-start-guide/)  
+- [Typegoose Documentation](https://typegoose.github.io/typegoose/docs/api/index-api)
+- [Quick start guide](https://typegoose.github.io/typegoose/docs/guides/quick-start-guide/)
```
Comment (hasezoey) [inline README.md:158]: why was this changed?

**Review by hasezoey — CHANGES_REQUESTED** [review]

**Review by hasezoey — APPROVED** [review]

**Comment by KrishnaMoorthy12** [comment]
Sorry that I did not make the suggested changes. My laptop died and I could not get it repaired because of lockdown.
Thanks for making the changes yourself.
